core-aam/index.html

```diff
@@ -548,18 +548,15 @@ <h2>Computed Role</h2>
 &lt;a href="#" role="button"&gt; &lt;!-- computedrole returns "button" --&gt;</pre
             >
           </aside>
-          <p>
-            When an element has a role but is not contained in the required context (for example, an orphaned `listitem` without the required accessible parent of role `list`), User Agents MUST ignore
-            the role token, and return the `computedrole` as if the ignored role token had not been included.
+          <p class="note">
+            When an element has a role but is not contained in the required context (for example, an orphaned `listitem` without the required accessible parent of role `list`), this is an authoring
+            error, but the user agent behavior is not specified as a single rule. For most roles, user agents can either recover the error by ignoring the role, or respect the author's intended role
+            in scenarios deemed by the implementation to be harmless. This unspecified permissiveness in how engines treat author role usage errors is occasionally overridden for a specific role in
+            other specs such as [[HTML-AAM]].
           </p>
           <aside class="example">
             <!-- ReSpec needs these examples to be unindented. -->
-            <pre>
-&lt;div role="listitem"&gt; &lt;!-- Author error: orphaned listitem. computedrole returns "generic" --&gt;
-
-&lt;div role="list"&gt; &lt;!-- computedrole returns "list" --&gt;
-  &lt;div role="listitem"&gt; &lt;!-- computedrole returns "listitem" in the required context. --&gt;</pre
-            >
+            <pre>&lt;div role="listitem"&gt; &lt;!-- Author error: orphaned listitem. computedrole is unspecified. --&gt;</pre>
```

Comment:
Actually, I think I should keep the second example.
```suggestion
            <pre>
&lt;div role="listitem"&gt; &lt;!-- Author error: orphaned listitem. computedrole is unspecified. --&gt;

&lt;div role="list"&gt; &lt;!-- computedrole returns "list" --&gt;
  &lt;div role="listitem"&gt; &lt;!-- computedrole returns "listitem" in the appropriate context. --&gt;</pre>
```